asia/malaysia/readme.md

```diff
@@ -0,0 +1 @@
+[Subang](./subang.md)
```

Comment:
This `readme.md` file should be following the same pattern in the filename as the one in `asia`, ie. be `README.md`

Also, it's good practice to have a title on the file.